### PR TITLE
Added 2 conditions to eventMacro , fixed another one.

### PR DESCRIPTION
**Pull request by Henrybk**
Added conditions InSaveMap and StorageOpened, fixed InLockMap to work with files realoading.

### CODE REVIEW

File: plugins/eventMacro/eventMacro/Condition/InLockMap.pm [hunk 1]
```diff
@@ -21,22 +21,39 @@ sub _parse_syntax {
 		return 0;
 	}
 
-	$self->{wanted_return_inLockMap} = $condition_code;
+	$self->{wanted_return_InLockMap} = $condition_code;
+	
+	$self->{lastMap} = $field ? $field->baseName : '';
```
Comment (allanon) [inline plugins/eventMacro/eventMacro/Condition/InLockMap.pm]: Is this identical to `$self->validate_condition('recheck')`?

File: plugins/eventMacro/eventMacro/Condition/InSaveMap.pm [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	my ( $self, $condition_code ) = @_;
+
+	if ( $condition_code !~ /^(0|1)$/ ) {
+		$self->{error} = "Value '$condition_code' should be '0' or '1'";
+		return 0;
+	}
+
+	$self->{wanted_return_InSaveMap} = $condition_code;
+	
+	$self->{lastMap} = $field ? $field->baseName : '';
```
Comment (allanon) [inline plugins/eventMacro/eventMacro/Condition/InSaveMap.pm]: Is this identical to $self->validate_condition('recheck')?

Comment (Henrybk) [inline plugins/eventMacro/eventMacro/Condition/InSaveMap.pm]: Yeah

File: plugins/eventMacro/eventMacro/Condition/InLockMap.pm (hunk 1 above)
Comment (Henrybk) [inline plugins/eventMacro/eventMacro/Condition/InLockMap.pm]: Yep

**Pull request merged**